deploy/helm/kubecf/templates/ingress.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.features.ingress.enabled -}}
+{{- if and .Values.features.ingress.tls.crt .Values.features.ingress.tls.key -}}
```

Comment:
If the secret is not created, how do we deal with https://github.com/SUSE/kubecf/pull/112/files#diff-f9706e030015368dbed3dcd52240b733R66-R69?

Reply:
The secret is going to be created by our cluster provider.